.github/workflows/pr-environment-update.yml

```diff
@@ -51,6 +55,71 @@ jobs:
           environment: ${{ inputs.environment }}
 
       - name: Update environment
+        id: update-environment
         run: ./bin/update-pr-environment "${{ inputs.app_name }}" "${{ inputs.environment }}" "${{ inputs.pr_number }}" "${{ inputs.commit_hash }}"
         env:
-          GH_TOKEN: ${{ github.token }}
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Capture service endpoint
+        id: get-service-endpoint
+        run: |
+            service_endpoint=$(terraform -chdir="infra/${{ inputs.app_name }}/service" output -raw service_endpoint)
+            echo "service_endpoint=${service_endpoint}"
+            echo "service_endpoint=${service_endpoint}" >> "$GITHUB_OUTPUT"
+
+  accessibility:
+    name: Accessibility Checks
+    needs: [update]
+    runs-on: ubuntu-latest
+
+    env:
+      SERVICE_ENDPOINT_URL: ${{ needs.update.outputs.service_endpoint }}
```

Comment:
looks like this job is a duplicate of the code in ci-app-accessibility-tests.yml. did you intend to call that workflow instead?

Reply:
Yep, I was trying to separate the accessibility job into its own file and was having trouble calling a separate workflow file. Know if it's possible? If so I think we should do it - could be a follow up PR .


If it's not possible I think we should just delete `ci-app-accessibility-test.yml`

Reply:
I know it's possible and think we should do it in this PR rather than as a separate PR. what issues did you run into?